config/opa-service/policy.rego

```diff
@@ -10,18 +10,41 @@ default authz = false
 # Action to method mappings
 scope_method := {"acl:Read": ["GET"], "acl:Write": ["POST"], "acl:Control": ["PUT", "DELETE"]}
 
-# Helper to get the token payload
-token = {"payload": payload} {
-  [header, payload, signature] := io.jwt.decode(bearer_token)
-}
-
 # Extract Bearer Token
 bearer_token = t {
 	v := authorization
 	startswith(v, "Bearer ")
 	t := substring(v, count("Bearer "), -1)
 }
 
+# Helper to get the token payload
+token = {"payload": payload, "header": header} {
+  [header, payload, signature] := io.jwt.decode(bearer_token)
+}
+
+# Token issuer
+issuer = token.payload.iss
```

Comment:
in this way you basically assume the issuer is the one in the token,
which means any token provided by any issuer will work out.

we should have an array of valid issuers as part of the opa data or
have some way to load it from evn variables, wdyt?